devvit.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://developers.reddit.com/schema/config-file.v1.json",
+  "name": "YOUR_APP_NAME",
```

Comment:
:no_bell: I think if you toss in an illegal character like "<YOUR_APP_NAME>", it'll get blocked by schema validation.